netket/operator/_kinetic.py

```diff
@@ -65,6 +72,12 @@ def logpsi_x(x):
 
         return -0.5 * jnp.sum(mass * (dp_dx2 + dp_dx), axis=-1)
 
+    @partial(jax.vmap, in_axes=(None, None, None, 0, None))
```

Comment:
this is a bit ugly: i need the batched kernel for the standard code-path, but i need the non-batched one to enable chunking (because we call `nkjax.vmap_chunked`). 

It's all internal, so I think it's fine for the time being, but eventually this should be cleaned up.

Probably, the easiest way to clean this stuff up is to make everything dataclasses, so we don't need those methods anymore...